select.scss

```diff
@@ -0,0 +1,127 @@
+// Select
+// ===
+
+
+// Configurable Variables
+// ---
+
+// Whether to draw the default down-arrow icon.
+$select__draw-caret: true !default;
+
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'spline/dist/utilities/triangle';
+@import 'level/level';
+
+
+// Root
+// ---
+
+.c-select {
+    position: relative; // Allow positioning of select element and caret
+
+    display: block;
+    box-sizing: border-box;
```

Comment:
Is there a reason we aren't adding this to level? I imagine we'll want to avoid declaring this on every stencil we make.